build/.htaccess

```diff
@@ -0,0 +1 @@
+Deny from all
```

Comment:
It's common for hosts to ignore .htaccess files that aren't in the root directory. What's the rationale for splitting this off?